bin/server/cmd/root.go

```diff
@@ -173,7 +192,30 @@ func start() {
 		return
 	}
 
-	activeProfile := activeProfile(flags.dataDir)
+	var profile server.Profile
+	if flags.backupBucket == "" {
```

Comment:
This amount of if-branching can be reduced when it's moved to getBaseProfile. Think think think of a way to simplify the code.